charts/localstack/templates/ingress.yaml

```diff
@@ -22,6 +22,10 @@ metadata:
     {{- tpl (toYaml .) $ | nindent 4 }}
     {{- end }}
 spec:
+{{- if .Values.ingress.ingressClassName }}
+  ingressClassName: {{ .Values.ingress.ingressClassName | quote }}
+{{- end }}
+
```

Comment:
nitpick: Please remove this newline here, the other elements also don't have newlines in between.